ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -45,6 +45,8 @@
   static final String ZKINFO_STR = "zkInfo";
   static final String DATACENTER_STR = "datacenter";
   static final String DATACENTER_ID_STR = "id";
+  static final String SCHEMA_VERSION_STR = "schemaVersion";
+  static final String XID_STR = "xid";
```

Comment:
not sure we should add this in this PR if we aren't going to use it.

Reply:
Will remove `XID` in this patch.